CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [0.2.1]
 
 * `Exporter` class exposes now `frames`.
+*  Bump `image` dependency to ^4.0.17
```

Comment:
This should be in a new release, not in an existing one. You also need to increase the version

Reply:
How so? `0.2.1` is not out yet. On pub.dev I only see `0.2.0`.

Reply:
Oh interesting, in that case it's okay I assume. Thanks for clarifying